helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -68,11 +68,17 @@ spec:
           command:
             - sh
             - '-c'
-            - 'cd /certs; for c in *; do
-                 keytool -import -alias "$c" -file "$c" -trustcacerts -noprompt
-                   -storepass changeit -keystore /etc/nexus-iq-server/jks/keystore.jks;
+            - 'cp -v /usr/lib/jvm/jre/lib/security/cacerts
+                 /etc/nexus-iq-server/jks/keystore.jks;
+               chmod 644 /etc/nexus-iq-server/jks/keystore.jks;
```

Comment:
this is new: cp and chmod.

Reply:
Does this mean that we won't need to add an outbound cert for other hosts like https://clm.sonatype.com/? Just like you did in our testing sessions (for instance to be able to upload the IQ license).